homeassistant/components/reolink/host.py

```diff
@@ -148,34 +148,43 @@ async def async_init(self) -> None:
 
         await self.subscribe()
 
-        _LOGGER.debug(
```

Comment:
Would it make sense to do `async_call_later` for 15s and check there instead of delaying init here?  (you would need to cancel the call later if the integration gets reloaded/unloaded before it fires)

Reply:
Thank you for the suggestion, that actually makes a lot of sense!
I implemented the changes in my latest commit.